misc/python/materialize/cloudtest/k8s/postgres_source.py

```diff
@@ -0,0 +1,81 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
```

Comment:
Given that the original `postgres.py` will need to eventually become `cockroach.py` , you were right in duplicating all that stuff.